helm/charts/hydra-maester/templates/deployment.yaml

```diff
@@ -43,6 +43,9 @@ spec:
             {{- if .Values.singleNamespaceMode }}
             - --namespace={{ .Release.Namespace }}
             {{- end }}
+            {{- if .Values.syncPeriod }}
+            - --sync-period={{ .Values.syncPeriod }}
```

Comment:
I have one issue here. The change isn't yet released, so the deployment will fail due to an unexpected input flag, as there is a default value, so this will always be set. 
Please either remove the redundant if, or the default value :)

Reply:
Good point, I'll remove the default value then.

This way users won't need to update their Values.yml file when they upgrade the chart.

Reply:
in that case please do it like this:

```yaml
# The minimum frequency at which watched resources are reconciled
syncPeriod: ""
# syncPeriod: 10h
```

Reply:
Done as you said.

Reply:
```suggestion
            - --sync-period={{ .Values.deployment.args.syncPeriod }}
```